### DISCUSSION TITLE
ref instead of useState on client side

### DISCUSSION THREAD

**Discussion by Gringo675** [nuxt/nuxt#16219] [Questions] [answered]
I suspect it’s stupid question, but please help resolve this.

To manage state, we should apply useState in composable modules, like `const useX = () => useState('x')` and never use `const state = ref()`. It’s understandable and concerns SSR rendering. But what if some pages in our app work only on client side? Can I manage this “private” state there via `ref()` (or rather `reactive()`) like in “ordinary” SPA app?  Say, in admin section of the app, that closed from server rendering in routeRules. Or is it can have some negative consequences, like performance issues, memory leakages, …? 

Thanks.

**Comment by wobsoriano** [comment, marked as answer]
You should be good
